### ISSUE TITLE
Provide multiple cc size (small, medium, large, extra-large) 

### ISSUE THREAD

**Issue by abadi199** [abadi199/elm-creditcard#5]
We can provide options in the model with `type Size = Small | Medium | Large | ExtraLarge`

**Comment by uebayasi** [comment]
If you mean that "cc size" == Credit Card image size, how about:

`<svg ... width="100%" height="100%" ...>`

?

Actually I need this to make the image size responsive.  I'm using Bootstrap and tagging the image as `.img-responsive`.

**Comment by abadi199** [comment]
Good idea. Maybe put that in `Config` as `responsiveSize` that can be either `True` or `False`? If you want to create a PR for it, I'll be happy to assist you and merge the code. 😃
